src/NServiceBus.RabbitMQ/Configuration/DelayedDeliverySettings.cs

```diff
@@ -21,7 +24,11 @@ public DelayedDeliverySettings DisableTimeoutManager()
         }
 
         /// <summary>
-        /// TODO: Fill this out
+        /// Tells the endpoint that all other endpoints also support native delays.
+        /// <para>
+        /// If this is called, the endpoint will no longer bind destination addresses to the delay
```

Comment:
I had to read this sentence 2 times to understand what it is about. What if we would write something as follows:
> If this is called, the endpoint will assume that all other endpoints use native delays and manage their own bindings to delay infrastructure. 

I don't know if this should be the place where we tell everyone that sending endpoint may create bdinging to the destination if this options is not set. We should describe it in doco, and keep this description simple. @bording what do you think?